tests/fileTypeTests.py

```diff
@@ -110,7 +110,7 @@ def test_program_mixed(self):
         """Test mixed job with some rows failing."""
         jobId = self.jobIdDict["programMixed"]
         self.passed = self.run_test(
-            jobId, 200, "finished", 21060, 4, "complete", 118, True)
+            jobId, 200, "finished", 20998, 4, "complete", 118, True)
```

Comment:
Okay, so we'll merge this PR as is to make these names available, and the next PR on this should include a way to report long names.  I think long-term we had wanted to report either short or long names based on whichever was provided in that file, but that may be part of another story.